test/helpers.py

```diff
@@ -80,6 +80,15 @@ def skip_windows(*args, **kwargs):
     return wrapper
 
 
+def slow_test(f):
+    @functools.wraps(f)
+    def wrapper(*args, **kwargs):
+        if 'CI' in os.environ and 'RUN_SLOW' not in os.environ:
+            raise unittest.SkipTest("Slow test skipped on CI run")
```

Comment:
It seems to run fast enough in some of the CI environments, so do you think we should turn it for at least one or two of the environments?  Since I develop on macOS I don't run the valgrind tests locally so I would like some assurance for the code that I'm submitting.

Reply:
Yeah, it's a tradeoff. Some of the schemes are simply too slow to run on Valgrind though...